include/uhr_func.hpp

```diff
@@ -721,7 +721,11 @@ void set_minute(uint8_t min, uint8_t &offsetH, uint8_t &voll) {
             offsetH = 1;
         } else {
             usedUhrType->show(viertel);
-            usedUhrType->show(nach);
+            if (G.UhrtypeDef == Uhr_125_Type2) {
+                usedUhrType->show(v_nach);
+            } else {
+                usedUhrType->show(nach);
+            }
```

Comment:
Wie wäre es, wenn wir hier immer `v_nach` nehmen, und die anderen Layouts `v_nach` und `nach` einfach immer gleich interpretieren?

Reply:
Das wäre auch eine Lösung. Mir sind allerdings die Vorteile dessen noch nicht so klar.

Reply:
Einziger Grund ist, dass man damit die Logik in die Layouts schieben kann, und hier kein `if` mehr braucht.
Wenn wir langfristig nachladbare Layouts haben wollen, dann sollten wir versuchen, in der Basis so wenig layoutspezifischen Code wie möglich zu haben.